pesacheck_meedan_bridge/py/main.py

```diff
@@ -6,13 +6,24 @@
 import settings
 from check_api import post_to_check
 from database import PesacheckDatabase, PesacheckFeed
+from lxml import etree
 from trafilatura import extract
 
 
 def html_to_text(content):
     return extract(content, include_links=True, include_images=True) or content
 
 
+def extract_summary(content):
+    tree = etree.Element("root")
+    tree.append(etree.HTML(content))
+    figures = tree.xpath("//figure")
+    if len(figures) == 0:
+        return None
+    h4 = figures[0].getprevious()
+    return h4.text
```

Comment:
If we don't need to confirm that it's `h4`, do we even need to name it `h4`?
```suggestion
    summary_el = figures[0].getprevious()
    return summary_el.text
```